lms/djangoapps/discussion/templates/discussion/discussion_board.html

```diff
@@ -58,8 +58,9 @@
     <header class="page-header has-secondary">
         ## Breadcrumb navigation
         <div class="page-header-main">
-            <div class="sr-is-focusable" tabindex="-1"></div>
-            <div class="has-breadcrumbs"></div>
+            <nav aria-label="${_('Forums')}" class="sr-is-focusable" tabindex="-1">
```

Comment:
Sorry for this, but just wondered whether this should say "Discussions" since we don't use "forums" elsewhere.